content/api/cli.md

```diff
@@ -180,11 +197,11 @@ These options allow you to manipulate optimisations for a production build using
 |---------------------------|--------------------------------------------------------|--------------------------------------|
 | --optimize-max-chunks     | Try to keep the chunk count below a limit              | LimitChunkCountPlugin                |
```

Comment:
These  plugins could use links too. If the pages don't exist, add dummy stubs so at least the links are right.